faqs/galaxy/histories_unshare.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Find Histories that have been shared with you, and unshare those not needed
+area: histories     
+box_type: tip        
+layout: faq        
+contributors: [nurzhamalyrys] 
+---
+
+1. All account Histories owned by others, but shared with you, can be reviewed under User > Histories shared with me.
+2. "Histories shared with me" is part of your own quota usage, and unsharing will not significantly reduce quota usage for most end-users. However, if there are hundreds (or more!), clearing these up periodically (unsharing) could be worth it.
+3. Unshare histories on this page using the pull-down menu per-history.
+
+Notes:
+- View is a review feature. The data cannot be worked with but many details, including tool and dataset metadata/parameters are included.
+- Copy those you want to work with. This will increase your quota usage since that history copy is now one of your histories, but that will also allow you manipulate the datasets or the history itself, independently from the original owner. All History/Dataset functions are available if the other person granted full access to the datasets to you. Dataset sharing is a distinct option on the Share or Publish form. Only the original History owner can set the level of sharing (History only, or also Datasets). If you need Dataset access and it wasn't granted, you might need to ask the other person to reshare with the box for "Also make all objects within the History accessible." checked. The other person can toggle sharing to "off" for review/changes, then toggled back to "on" after.
```

Comment:
```suggestion
- Copy those you want to work with. This will increase your quota usage since that history copy is now one of your histories, but that will also allow you to manipulate the datasets or the history itself, independently from the original owner. All History/Dataset functions are available if the other person granted full access to the datasets to you. Dataset sharing is a distinct option on the Share or Publish form. Only the original History owner can set the level of sharing (History only, or also Datasets). If you need Dataset access and it wasn't granted, you might need to ask the other person to reshare with the box for "Also make all objects within the History accessible." checked. The other person can toggle sharing to "off" for review/changes, then toggled back to "on" after.
```